beast/physicsmodel/stars/isochrone.py

```diff
@@ -342,7 +368,8 @@ def interpolation(self, b=None):
             return self.interp
 
     def _load_table_(self, source):
-        self.data = Table(self.source).selectWhere("*", "isfinite(logA)")
+        tdata = Table.read(self.source, format="csv", header_start=_get_num_comments(self.source))
```

Comment:
Can we use 'comment' parameter here instead of using 'get_num_comments' function?

Reply:
Can you be more explicit in this suggestion?  I couldn't figure out how to tell the Table.read function where to get the header column names for the specific format of the saved isochrone files.  So I created this the `_get_num_comments` function to determine the number of comment lines in the file.  I tried other ways, but only got this to work.  Happy to do this a different way if there is one.

Reply:
For an isochrone file, 
tdata = Table.read('beast_example_phat_iso.csv', format="csv", comment='#') 
gives me the tdata.colnames as follows:
['logL', 'logg', 'mbolmag', 'logA', 'logT', 'M_ini', 'M_act', 'stage', 'Z']
Is this doing what you want?

Reply:
Awesome!  Most excellent.  Thanks.  I've updated the code, removed my `_get_num_comments` hack and am re-running all the tests.  Yeah!